packages/gamut-labs/src/AppHeader/index.tsx

```diff
@@ -84,16 +88,46 @@ export const mapItemToElement = (
         <AppHeaderDropdown onKeyDown={onKeyDown} action={action} item={item} />
       );
     case 'catalog-dropdown':
+      const catalogItemCount = languageAndSubjectCount(item);
       return (
-        <AppHeaderCatalogDropdown action={action} item={item} isAnon={isAnon} />
+        <AppHeaderDropdownSectionContainer
+          action={action}
+          item={item}
+          dataFocusable="[data-focusablecatalog=true]"
+          itemsCount={catalogItemCount}
```

Comment:
nit: if were just using `catalogItemCount` here and not any place else, id say we can just do 
```suggestion
          itemsCount={languageAndSubjectCount(item)}
```